### PR TITLE
Bump `tokio` to 1.17.0

### PR DESCRIPTION
**Pull request by koute**
Companion for https://github.com/paritytech/substrate/pull/10894

### CODE REVIEW

**Review by KiChjang — APPROVED** [review]

File: bridges/relays/client-substrate/Cargo.toml [hunk 1]
```diff
@@ -14,7 +14,7 @@ jsonrpsee-ws-client = "0.3.1"
 log = "0.4.11"
 num-traits = "0.2"
 rand = "0.7"
-tokio = "1.8"
+tokio = "1.17.0"
```
Comment (bkchr) [inline bridges/relays/client-substrate/Cargo.toml:17]:
```suggestion
tokio = "1.8"
```

These kind of unnecessary changes should be done in the bridges repo.

**Review by bkchr — APPROVED** [review]

File: bridges/relays/client-substrate/Cargo.toml (hunk 1 above)
Comment (wigy-opensource-developer) [reply bridges/relays/client-substrate/Cargo.toml:17]: If we have code here that is not needed, why do not we delete the whole folder?
